dashboard/app/controllers/levels_controller.rb

```diff
@@ -43,7 +43,7 @@ def edit_blocks
     @level = Level.find(params[:level_id])
     authorize! :edit, @level
     type = params[:type]
-    blocks_xml = @level.properties[type].presence || @level[type]
+    blocks_xml = @level.properties[type].presence || @level[type] || '<xml></xml>'
```

Comment:
Looks like your xml bits are being stripped from your comments, so im not 100% sure what you're asking